index.md

```diff
@@ -11,6 +11,8 @@ This is a keyboard firmware based on the [tmk\_keyboard firmware](http://github.
 * [ErgoDox EZ](https://github.com/qmk/qmk_firmware/blob/master/keyboards/ergodox_ez/)
 * [Clueboard](https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/)
 * [Cluepad](https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/17/)
+* [Gergo](/gergo/)
```

Comment:
Just FYI, these links on Lines 14-15 will only work if viewed from [qmk.fm](https://qmk.fm/); they don't work if viewed from GitHub. You could make them direct links though, for example:

```suggestion
* [Gergo](https://qmk.fm/keyboards/gergo/)
```